src/main/java/uk/org/tombolo/importer/ons/ONSWellbeingImporter.java

```diff
@@ -0,0 +1,160 @@
+            Sheet datatypeSheet = workbook.getSheetAt(sheet);
+
+            // Creating the row iterator object
+            Iterator<Row> rowIterator = datatypeSheet.rowIterator();
+            // Skipping unrelevant rows
+            int ignore = 0;
+            while (ignore++ < 7) {
+                rowIterator.next();
+            }
+
+            // Looping through rows
+            while (rowIterator.hasNext()) {
+                Row row = rowIterator.next();
+                String geograghy =  String.valueOf(row.getCell(0)).trim();
+                Subject subject = SubjectUtils.getSubjectByTypeAndLabel(localauthority, geograghy);
+                subject = subject != null ? subject : SubjectUtils.getSubjectByTypeAndLabel(englandboundaries, geograghy);
+
+                // Checking if subject is null
+                if (subject != null) {
+
+                    // Looping through the time values
+                    for (int timeValuesIndex=2; timeValuesIndex <= 7; timeValuesIndex++ ) {
+                        // This is the row number that contains our time values (years) in the dataset
+                        Row rowTime = datatypeSheet.getRow(5);
+                        String year = rowTime.getCell(timeValuesIndex).toString();
+                        year = year.substring(0, year.length() - 3);
+
+                        LocalDateTime timestamp = TimedValueUtils.parseTimestampString(year);
+
+                        try {
+                            Double record = row.getCell(timeValuesIndex).getNumericCellValue();
+                            // Here is where we are assigning the values of our .xls file to the attribute fields we
+                            // created.
+                            Attribute attribute = datasource.getTimedValueAttributes().get(sheetIndex);
+                            timedValues.add(new TimedValue(
+                                    subject,
+                                    attribute,
+                                    timestamp,
+                                    record));
+                        } catch (IllegalStateException e) {
+                            log.warn("Value for subject " + subject.getLabel() + " not found. " +
+                                    "Defaulting to 0.0. Consider using a BackoffField or ConstantField.");
+                            continue;
+                        }
+                    }
+                }
+            }
+            sheetIndex++;
+        }
+        saveAndClearTimedValueBuffer(timedValues);
+    }
+
+    @Override
+    public List<Attribute> getTimedValueAttributes(String datasourceID) {
+        // Creating a placeholder for our attributes
+        List<Attribute> attributes = new ArrayList<>();
+
+        // Dataset specific: we hardcode the columns names for the our .csv file
+        String[] elements = { "life_satisfaction", "worthwhile", "happiness", "anxiety"};
```

Comment:
you might want to consider adding the word `mean` in the labels and description.